src/Components/MailtoDropdown.js

```diff
@@ -24,6 +31,77 @@ export const MAILTO_HANDLERS = {
     yahoo: {
         link: d => `https://compose.mail.yahoo.com/?to=${d.email}&subject=${d.subject}&body=${d.body}`,
         countries: ['all']
+    },
+    copymanually: {
+        onClick: d => {
+            const dismiss = () => preact.render('', document.body, modal);
+            const onInputClick = e => {
+                if (previous_active_element.id === e.target.id) return;
+
+                e.target.select();
+                e.target.focus();
+                previous_active_element = e.target;
+            };
+            const modal = preact.render(
+                <IntlProvider scope="generator" definition={I18N_DEFINITION}>
+                    <Modal
+                        positiveText={<Text id="ok" />}
+                        onPositiveFeedback={dismiss}
+                        positiveDefault={true}
+                        onDismiss={dismiss}>
+                        <Text id="copymanually-explanation" />
+
+                        <div className="form-group">
+                            <strong>
+                                <label htmlFor="mailto-dropdown-copymanually-subject">
+                                    {t('subject', 'generator')}
+                                </label>
+                            </strong>
+                            <input
+                                type="text"
+                                id="mailto-dropdown-copymanually-subject"
+                                className="form-element"
+                                value={decodeURIComponent(d.subject)}
+                                onClick={onInputClick}
+                                readOnly
+                            />
+
+                            {d.email
```

Comment:
I actually think conditionally hiding this is confusing. Why not just leave it empty? When I tested I was cunfused whether it was missing. An empty input would hint to users that they need to select a recipient.

Reply:
Hm, interesting. Personally I think showing the empty input looks quite stupid.

But I mean, eh. It's not the end of the world. Sure, why note.